en/docs/learn/rate-limiting/introducing-throttling-use-cases.md

```diff
@@ -27,19 +27,19 @@ The following sections describe the type of throttling policy applicable to each
 
 #### Maximum backend throughput ( **Applies per API** ): API Publisher
 
-According to the API Gateway architecture, an API in the Gateway is actually a proxy to an actual service hosted within your organization, cloud, etc. This usually means that there is a physical capacity that your backend services can handle. Although you expose your API on defined limits (subscription tiers), as the number of applications that consume your API grows, the number of requests being served by your API rise, which in turn means that the number of requests served by your backend system rise as well. Therefore, although none of the applications may exceed their own allocated quotas, their combined load might hit the maximum capacity that can be handled by your backend system. To prevent your backend system from getting overloaded, the limits enforced by the **Maximum Backend Throughput** in the API act as a hard stop on the number of requests that your backend system can serve within a given time period. The counters maintained when evaluating the maximum backend throughput are shared across all nodes of the Gateway cluster and apply across all users using any application that accesses that particular API. For information on how to specify maximum backend throughput limits, see [Setting Maximum Backend Throughput Limits](../setting-maximum-backend-throughput-limits) .
+According to the API Gateway architecture, an API in the Gateway is actually a proxy to an actual service hosted within your organization, cloud, etc. This usually means that there is a physical capacity that your backend services can handle. Although you expose your API on defined limits (subscription tiers), as the number of applications that consume your API grows, the number of requests being served by your API rise, which in turn means that the number of requests served by your backend system rise as well. Therefore, although none of the applications may exceed their own allocated quotas, their combined load might hit the maximum capacity that can be handled by your backend system. To prevent your backend system from getting overloaded, the limits enforced by the **Maximum Backend Throughput** in the API act as a hard stop on the number of requests that your backend system can serve within a given time period. The counters maintained when evaluating the maximum backend throughput are shared across all nodes of the Gateway cluster and apply across all users using any application that accesses that particular API. For information on how to specify maximum backend throughput limits, see [Setting Maximum Backend Throughput Limits]({{base_path}}/learn/rate-limiting/setting-maximum-backend-throughput-limits).
```

Comment:
Break this into 2 paragraphs with "To prevent your backend system from getting overloaded"

Reply:
Fixed with 8ac6250